### ISSUE TITLE
Any recommendations on using the parameters?

### ISSUE THREAD

**Issue by valentinvieriu** [rtlee9/Trump-bot#3]
Do you have any tips on how to play with the `train.lua` parameters? Just looking ate the readme seems that it's like somehow of magic.

Any feedback is appreciated!

**Comment by rtlee9** [comment]
For tuning, I spent most of my time searching for good values of `rnn_size`, `num_layers`, `learning_rate`, and `dropout` by making small changes one at a time and seeing what the impact was on training and cross validation performance. Generally, you'll want to tweak these hyperparameters to minimize your cross validation loss, while making sure your training loss isn't significantly lower (which would be indicative of overfitting). A more complete but involved approach would be to implement a grid search or a random search for an optimal set of hyperparameters.
